docs/generators/python-aiohttp.md

```diff
@@ -14,6 +14,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |ensureUniqueParams|Whether to ensure parameter names are unique in an operation (rename parameters that are not).| |true|
 |featureCORS|use flask-cors for handling CORS requests| |false|
 |legacyDiscriminatorBehavior|Set to false for generators with better support for discriminators. (Python, Java, Go, PowerShell, C#have this enabled by default).|<dl><dt>**true**</dt><dd>The mapping in the discriminator includes descendent schemas that allOf inherit from self and the discriminator mapping schemas in the OAS document.</dd><dt>**false**</dt><dd>The mapping in the discriminator includes any descendent schemas that allOf inherit from self, any oneOf schemas, any anyOf schemas, any x-discriminator-values, and the discriminator mapping schemas in the OAS document AND Codegen validates that oneOf and anyOf schemas contain the required discriminator and throws an error if the discriminator is missing.</dd></dl>|true|
+|moveTestsPythonSrcRoot|generates test under the pythonSrcRoot folder.| |false|
```

Comment:
Why add a new CLI flag here?
In my opinion if pythonSrcRoot is set, then the sets should use it too.
Also do the tests use the right import path when testing?
If pythonSrcRoot is set, then the imports will change, right?

Reply:
I've wrapped this with another CLI parameter to prevent this new convention
breaking folks that are already using pythonSrcRoot and have built things
around how it currently functions. The other CLI parameter
"usePythonSrcRootInImports"  modifies the actual imports. I've covered this
in my unit test. I could generate some samples using these parameters to highlight all of this.

Thoughts?